pkg/systemd/services/service-details.jsx

```diff
@@ -473,7 +473,7 @@ export class ServiceDetails extends React.Component {
                             <DescriptionListTerm>{rel.Name}</DescriptionListTerm>
                             <DescriptionListDescription id={rel.Name.split(" ").join("")}>
                                 <ul className="comma-list">
-                                    {rel.Units.map(unit => <li key={unit}><Button isInline variant="link" component="a" href={"#/" + unit} isDisabled={!this.props.isValid(unit)}>{unit}</Button></li>)}
+                                    {rel.Units.map(unit => <li key={unit}><Button isInline variant="link" component="a" href={"#/" + unit + (this.props.owner === "user" ? "?owner=user" : "")} isDisabled={!this.props.isValid(unit)}>{unit}</Button></li>)}
```

Comment:
This looks the wrong way around -- every [documentation](https://datatracker.ietf.org/doc/html/rfc3986/#page-16) [that I](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax) [know of](https://www.geeksforgeeks.org/components-of-a-url/) has the query *first* and then the fragment. Apparently cockpit-ws is cool with that order, but maybe let's stick to the common format? Or do we use this order somewhere else already?

Reply:
this is how we do it everywhere. For example just to open user service: `https://localhost:9090/system/services#/colord-session.service?owner=user` but even logs `https://localhost:9090/system/logs#/s=19a35472f0f24e9cba7fd302491e93ab;i=1b0d2dd;b=6996acaedd1841efb0202962dcd37c38;m=436ac49485;t=5d6f0f1ae5216;x=9de9d67f897c7222?parent_options={%22priority%22:%22err%22}`

Reply:
Fair enough.